tests/cross_functional/system_test/test_rbd_encrypted_pvc_keyrotation_system.py

```
@@ -0,0 +1,509 @@
+            pod_objs (list): List of encrypted RBD pod objects
+            pvc_combinations (list): List of PVC configurations for encrypted RBD
+            non_enc_pod_objs (list): List of non-encrypted RBD pod objects
+            non_enc_combinations (list): List of PVC configurations for non-encrypted RBD
+            cephfs_pod_objs (list): List of CephFS pod objects
+            nfs_pod_objs (list): List of NFS pod objects
+
+        """
+        # Prepare pods with config for FIO
+        fs_config = {"volume_mode": constants.VOLUME_MODE_FILESYSTEM}
+        all_pods_with_config = (
+            list(zip(pod_objs or [], pvc_combinations or []))
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)